css/components/button.css

```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap');
+
+
+:root{
+    --lightBlack: #303030;
+    --darkBlack: #000;
+    --white: #fff;
+    --offWhite: #F5F3FF;
+    --navyBlue: #082032;
```

Comment:
I suggest to avoid using value of te variable as a part of the variable name. Reason being if you change the color, then you'd have to change it everywhere which beats the purpose of using a variable.